src/extension.ts

```diff
@@ -173,26 +173,13 @@ export async function activate(
               return;
             }
 
-            const exists = async (filename: string) => {
-              try {
-                await vscode.workspace.fs.stat(
-                  vscode.Uri.joinPath(workspaceUri, filename)
-                );
-                return true;
-              } catch {
-                return false;
-              }
-            };
-
-            const command =
-              vscode.workspace
-                .getConfiguration("storybook-opener.storybookOption")
-                .get("startCommand") ?? //
-              (await exists("pnpm-lock.yaml"))
-                ? "pnpm run storybook --no-open"
-                : (await exists("yarn.lock"))
-                ? "yarn storybook --no-open"
-                : "npm run storybook --no-open";
+            const config = vscode.workspace.getConfiguration(
+              "storybook-opener.storybookOption"
+            );
+            const port = config.get<number>("port");
+            const startCommand = config.get<string>("startCommand");
+
+            const command = startCommand || `npx storybook dev -p ${port} --no-open`;
```

Comment:
`-p ${port}` Nice. Can you add the other options (`host`, `https`)?

Reply:
@ygkn 
Thanks for your review.
When localhost is entered in the `host` option, the storybook did not open, so it is set to an empty string.

[5e6be55](https://github.com/ygkn/storybook-opener/pull/6/commits/5e6be55e8d06181fad87b1eb5d83d409879be8b6)

Reply:
cool!